src/User.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const User = ({ user }) => (
+  <td>{user.name}</td>
```

Comment:
У тебя td будет внутри td
```suggestion
  {user.name}
```
Или убери внешнюю td в компоненте TodoItem